android/java/org/chromium/chrome/browser/ntp/IncognitoNewTabPageView.java

```diff
@@ -85,21 +85,21 @@ protected void onFinishInflate() {
         // any shortcut causes the UrlBar to be focused. See ViewRootImpl.leaveTouchMode().
         mScrollView.setDescendantFocusability(FOCUS_BEFORE_DESCENDANTS);
 
-        mVpnCta = findViewById(R.id.tv_try_vpn);
-        if (BraveVpnUtils.isVpnFeatureSupported(getContext())
```

Comment:
if we just added a false in this If block then then else block would have hidden this as well? and the number of changes would become minimum which reduces the chances of conflicts later on